packages/fetchai/skills/tac_negotiation/behaviours.py

```diff
@@ -131,7 +131,7 @@ def _register_service(self) -> None:
                 )
             )
             goods_supplied_description = strategy.get_own_service_description(
-                is_supply=True
+                is_supply=True, is_search_description=False,
```

Comment:
Annoyingly, the OEF cannot deal with attribute names made up of a string of 0-9. Hence, we need to create slightly different attribute names just for searching. Lame but this is a workaround which does the trick for now.